.github/workflows/release.yml

```diff
@@ -67,5 +67,5 @@ jobs:
           AWSUI_GITHUB_BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
           AWSUI_GITHUB_COMMIT_SHA: ${{ github.event.commits[0].sha }}
           AWSUI_GITHUB_COMMIT_MESSAGE: ${{ github.event.commits[0].message }}
-          AWSUI_GITHUB_REPOSITORY_NAME: ${{ github.event.repository.name }}
+          AWSUI_GITHUB_REPOSITORY_NAME: ${{ github.event.repo.name }}
```

Comment:
https://docs.github.com/en/rest/using-the-rest-api/github-event-types?apiVersion=2022-11-28